packages/now-cli/test/dev/fixtures/30-next-image-optimization/vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": 2,
+  "build": {
+    "env": {
+      "FORCE_BUILDER_TAG": "canary"
```

Comment:
Why is this necessary?

Reply:
Its not necessary but it will help us catch differences in canary before they go to stable